### ISSUE TITLE
.info, Indexes, ETFs, mutual funds, currencies not working in latest release (recommend 0.1.45 until fixed)

### ISSUE THREAD

**Issue by GregoryMorse** [ranaroussi/yfinance#126]
This release (and all >= 0.1.46) are still problematic - as all this functionality worked perfectly in the prior one.  For an index '^GSPC':
```
  File "C:\ProgramData\Anaconda3\lib\site-packages\yfinance\base.py", line 279, in _get_fundamentals
    self._info = data['summaryProfile']

KeyError: 'summaryProfile'
```

For an ETF 'IWO':

```
TypeError: 'NoneType' object is not iterable
… #info returns None
File "C:\ProgramData\Anaconda3\lib\site-packages\yfinance\ticker.py", line 122, in info
    return self.get_info()
```
For mutual fund I am getting
```
File "C:\ProgramData\Anaconda3\lib\site-packages\yfinance\ticker.py", line 122, in info
    return self.get_info()
ValueError: If using all scalar values, you must pass an index
```

For currencies I get with 'BTC-USD' an empty dictionary missing values like the most important `KeyError: 'quoteType'`.  Another issue reported similarly saying the dictionary is empty now and does not include the normal information.

It could be because yahoo seems to be doing intensive updates to their backend and frontend of the site.  There is no good solution except to try to keep up with the changes and the maintainer continue making releases which deal with this.  These also is likely because the testing is not enough.

I ask the maintainer to make sure to test the following 5 very important test cases per release - probably writing a script to do it (can make a PR for this test function if asked):
index e.g. S&P500 '^GSPC'
mutual fund e.g. Vanguard 500 Index Fund 'VFINX'
ETF e.g. Russell 2000 Growth 'IWO'
security e.g. Microsoft 'MSFT' (probably what is being tested currently)
currency e.g. BitCoin-US dollars 'BTC-USD' (as original post gives)

These things all worked in the prior release to the current one.  More logic needed as obviously the summaryProfile is not available in an index but the info is.  The idea is to return as much info as possible like the old functionality.  The info seems to be totally broken in the latest version for even normal securities.

**Comment by kennguyen01** [comment]
Same here, my project on the laptop using an older version worked while the newest version on the PC didn't.

**Comment by ranaroussi** [comment]
Fixed in `0.1.50`. Please re-open if the problem persists.
